Makefile

```diff
@@ -95,15 +95,15 @@ e2e-test:
 e2e-bootstrap: install-helm
 	docker pull $(IMAGE_TAG) || make container-all push-manifest
 ifdef TEST_WINDOWS
-	helm install csi-driver-smb charts/latest/csi-driver-smb --namespace kube-system --wait --timeout=15m -v=5 --debug \
+	helm upgrade csi-driver-smb charts/latest/csi-driver-smb --namespace kube-system --wait --timeout=15m -v=5 --debug --install \

```

Comment:
I found a bug for `helm upgrade`, agent node will always use `v0.7.0` in azure pipeline which breaks the pipeline finally, I would revert to use original `helm instal`
```
print out csi-smb-node logs ...
======================================================================================
[pod/csi-smb-node-sqjc2/smb] I0526 01:09:40.294321       1 main.go:85] set up prometheus server on [::]:29645
[pod/csi-smb-node-sqjc2/smb] I0526 01:09:40.294699       1 smb.go:62] 
[pod/csi-smb-node-sqjc2/smb] DRIVER INFORMATION:
[pod/csi-smb-node-sqjc2/smb] -------------------
[pod/csi-smb-node-sqjc2/smb] Build Date: "2021-05-22T01:05:27Z"
[pod/csi-smb-node-sqjc2/smb] Compiler: gc
[pod/csi-smb-node-sqjc2/smb] Driver Name: smb.csi.k8s.io
[pod/csi-smb-node-sqjc2/smb] Driver Version: v0.7.0
[pod/csi-smb-node-sqjc2/smb] Git Commit: a03a482b5e925a77edcdfbab2ae330be8f5866ff
[pod/csi-smb-node-sqjc2/smb] Go Version: go1.16
[pod/csi-smb-node-sqjc2/smb] Platform: linux/amd64
```

https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-csi_csi-driver-smb/265/pull-csi-driver-smb-e2e-windows/1397354833710682112/build-log.txt

Reply:
nvm, it's not related, still try to figure out why agent node is always pulling image `v0.7.0`